src/components/common/NameRequestInfo.vue

```diff
@@ -101,7 +99,8 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin, NameRe
 
   /** Return formatted expiration date */
   private formattedExpirationDate (): string {
-    return this.formatDateString(this.nameRequestDetails.expirationDate)
```

Comment:
This was a bug: `formatDateString()` didn't like the expirationDate format and returned null. Now fixed and consistent with other date strings (eg, "Nov 23, 2021").